receiver/hostmetricsreceiver/example_config.yaml

```diff
@@ -4,7 +4,7 @@ extensions:
 
 receivers:
   hostmetrics:
-    default_collection_interval: 60s
+    collection_interval: 1m
     scrapers:
       cpu:
         report_per_cpu: true
```

Comment:
I would keep this as false in the example because people will copy paste.

Reply:
I changed the default to be false, and removed this line from the example config